src/main/java/com/disraely/entities/BaseEntity.java

```diff
@@ -0,0 +1,35 @@
+package com.disraely.entities;
+
+import java.io.Serializable;
+
+public class BaseEntity implements Serializable {
+    private Long id;
+    private Long createDate;
```

Comment:
Because I want to store all dates in the system as milliseconds. The date type in java is the most complex of what I need. Saving dates in milliseconds (long) can also be more efficient for mathematical operations or sending them to another system to be interpreted more easily.

Reply:
This is not true, I think by doing this you will have to deal with things that java developers have already faced and it handled by the jdk. By doing this you will make your code even harder to read and more complex. Just think that your code will need to make conversion on every operation. It could cost one or 2 lines that you can save or creating unnecessary utilities function. If you don't have a use case right now, please use what jdk offers

Reply:
Entendido 🙌